relay.go

```diff
@@ -8,6 +8,13 @@ import (
 	"time"
 )
 
+// _maxRelayTombs is the maximum number of tombs we'll accumulate in a single
+// relayItems.
+const _maxRelayTombs = 1 << 20
+
+// _relayTombTTL is the length of time we'll keep a tomb before GC'ing it.
+const _relayTombTTL = 10 * time.Minute
```

Comment:
Sure, works for me. Since I was picking random numbers, I thought I'd start by erring on the high side.